packages/docusaurus-theme-classic/package.json

```diff
@@ -7,6 +7,11 @@
     "access": "public"
   },
   "license": "MIT",
+  "scripts": {
+    "tsc": "tsc --noEmit && yarn babel && yarn prettier",
```

Comment:
We should probably rename the tsc scripts to just `build` or something (can be in another PR)

Also, I added recently a global `yarn watch` command, all TS subprojects should rather implement it as well